api/clients/buckets/bucket.go

```diff
@@ -32,22 +32,18 @@ import (
 
 const endpointPath = "platform/storage/management/v1/bucket-definitions"
 
-type Response struct {
-	api.Response
-}
-
 type response struct {
 	api.Response
 	BucketName string `json:"bucketName"`
 	Status     string `json:"status"`
 	Version    int    `json:"version"`
 }
 
+type Response = api.Response
```

Comment:
Is it possible to remove the alias also?
I don't see added value

Reply:
The advantage is, that you don't need to import two packages but just e.g. "github.com/dynatrace/dynatrace-configuration-as-code-core/api/clients/buckets" when using that client.